services/user/delete.go

```diff
@@ -1,8 +1,7 @@
-// Copyright 2014 The Gogs Authors. All rights reserved.
-// Copyright 2019 The Gitea Authors. All rights reserved.
+// Copyright 2023 The Gitea Authors. All rights reserved.
 // SPDX-License-Identifier: MIT
 
-package models
+package user
```

Comment:
What's the benefit of extracting it from `models` into `services`?
That there is no more user deletion logic inside `models`?

Reply:
Currently, because of dependency cycle problem, it has to be put in `models` but not `models/user` package. My ideal idea is to move almost files/functions from `models` to other places.